### DISCUSSION TITLE
Account Suspension

### DISCUSSION THREAD

**Discussion by felipe-sartes** [pear-devs/pear-desktop#598] [Q&A] [answered]
Its possible my account being suspended using yt music with adblock on desktop or its totally safe? Im discover this app in the last hour so sorry if i'm being dumb or newbie

**Comment by Araxeus** [comment, marked as answer]
100% safe

**Reply by cggjaicf** [reply]
would probably be nice to have this addressed in README with some explanation or confirmation on why this isn't bannable cuz I also ask myself if I'll get suspended every time I open this app lol
